src/ModelContextProtocol/Server/McpServer.cs

```diff
@@ -396,13 +396,12 @@ private void SetToolsHandler(McpServerOptions options)
                 foreach (McpServerTool tool in tools)
                 {
                     resultTools.Add(tool.ProtocolTool);
-                    if(progressToken is not null)
+                    if (progressToken is not null)
                     {
                         await this.NotifyProgressAsync(new()
                         {
                             Progress = resultTools.Count,
                             Total = totalProgress,
-                            Message = $"Listing tool '{tool.ProtocolTool.Name}'...",
                         }, cancellationToken).ConfigureAwait(false);
                     }
```

Comment:
Let's delete this whole section. I'm not seeing the benefit it adds when there's no custom handler.